src/Streamline.cpp

```diff
@@ -170,41 +170,39 @@ HRESULT Streamline::CreateDeviceAndSwapChain(IDXGIAdapter* pAdapter,
 		}
 	}
 
-	if (!REL::Module::IsVR()) {
-		slIsFeatureLoaded(sl::kFeatureDLSS_G, featureDLSSG);
-		if (featureDLSSG) {
-			logger::info("[Streamline] DLSSG feature is loaded");
-			featureDLSSG = slIsFeatureSupported(sl::kFeatureDLSS_G, adapterInfo) == sl::Result::eOk;
-		} else {
-			logger::info("[Streamline] DLSSG feature is not loaded");
-			sl::FeatureRequirements featureRequirements;
-			sl::Result result = slGetFeatureRequirements(sl::kFeatureDLSS_G, featureRequirements);
-			if (result != sl::Result::eOk) {
-				logger::info("[Streamline] DLSSG feature failed to load due to: {}", magic_enum::enum_name(result));
-			}
+	slIsFeatureLoaded(sl::kFeatureDLSS_G, featureDLSSG);
```

Comment:
instead of adding 10 extra `&& !REL::Module::IsVR()` to the if statements, can't you set this bool to false?

Reply:
Nope, tried assigning to `featureDLSSG` based off of vr setting and it gets reverted immediately. I think it's controlled by the streamline driver or something.

Reply:
Even my updated comment? Don't we control that boolean?

Reply:
I didn't do your specific change but I've tried to edit it twice and done two work arounds. First was to use enable settings and control that Boolean. Per doodlum's dislike of settings, he removed it and broke VR so hard coded us out. This is the second attempt to just change it manually everywhere.